perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -303,6 +318,7 @@ bool OccupancyMapMonitor::loadMapCallback(moveit_msgs::LoadMap::Request& request
   tree_->lockWrite();
   try
   {
+    tree_->enableChangeDetection(false);
```

Comment:
After reviewing ros-planning/moveit_ros#602 (now ros-planning/moveit_ros#636) I believe that this line should not be here (in this PR), but here should actually go to the load via param in line 102.